### PR TITLE
Adding the variable `KNATIVE_EVENTING_RELEASE_TLS` to install latest eventing TLS

### PR DESCRIPTION
**Pull request by Leo6Leo**
# Changes

- Adding the KNATIVE_EVENTING_RELEASE_TLS

**Release Note**

```release-note
Adding the option to install knative eventing TLS component
```

**Docs**

```docs

```

### CODE REVIEW

File: library.sh [hunk 1]
```diff
@@ -1031,3 +1031,4 @@ readonly KNATIVE_SERVING_RELEASE_CRDS="$(get_latest_knative_yaml_source "serving
 readonly KNATIVE_SERVING_RELEASE_CORE="$(get_latest_knative_yaml_source "serving" "serving-core")"
 readonly KNATIVE_NET_ISTIO_RELEASE="$(get_latest_knative_yaml_source "net-istio" "net-istio")"
 readonly KNATIVE_EVENTING_RELEASE="$(get_latest_knative_yaml_source "eventing" "eventing")"
+readonly KNATIVE_EVENTING_RELEASE_TLS="$(get_latest_knative_yaml_source "eventing" "eventing-tls-networking")"
```
Comment (pierDipi) [inline library.sh:1034]:
```suggestion
readonly KNATIVE_EVENTING_RELEASE_TLS="$(get_latest_knative_yaml_source "eventing" "eventing-tls-networking")"

```

**Comment by pierDipi** [comment]
One comment

/approve

**Review by Cali0707 — APPROVED** [review]
/lgtm

**Pull request merged**